django_airavata/apps/admin/static/django_airavata_admin/src/components/admin/group_resource_preferences/GroupComputeResourcePreference.vue

```diff
@@ -141,8 +174,18 @@ export default {
     id: {
       type: String,
     },
+    readonly: {
+      type: Boolean,
```

Comment:
If the type can also be string, then you can have `type: [Boolean, String]` so both are accepted.